src/software/thunderscope/log/g3log_checkboxes.py

```diff
@@ -0,0 +1,49 @@
+import pyqtgraph as pg
+from PyQt5.QtWidgets import *
+
+
+class g3logCheckboxes(QWidget):
+    def __init__(self):
+        QWidget.__init__(self)
+        layout = QGridLayout()
+        self.setLayout(layout)
+
+        # Creates 4 checkboxes based on the 4 log types
+        self.cbutton1 = QCheckBox("DEBUG")
+        self.cbutton1.setChecked(True)
+        self.cbutton1.log_level = "DEBUG"
+        layout.addWidget(self.cbutton1, 0, 0)
+
+        self.cbutton2 = QCheckBox("INFO")
+        self.cbutton2.setChecked(True)
+        self.cbutton2.log_level = "INFO"
+        layout.addWidget(self.cbutton2, 1, 0)
```

Comment:
i think we want (0,1) (0,2) (0,3) ... so that the checkboxes are horizontal rather than vertical